### PR TITLE
Typo in "dive into code"

### PR DESCRIPTION
**Pull request by glikson**

### CODE REVIEW

File: shared/diveIntoCode/go.md [hunk 1]
```diff
@@ -53,4 +53,4 @@ CMD ./resin-go-hello-world
 [resinDockerHub]:https://hub.docker.com/u/resin/
 [docker-ref]:https://docs.docker.com/engine/reference/builder/
 [systemd-link]:https://en.wikipedia.org/wiki/Systemd
-[listOfResinNames]:/devicetypes/
\ No newline at end of file
+[listOfResinNames]:/devicetypes/
```
Comment (emirotin) [inline shared/diveIntoCode/go.md]: What is the change on this line? A new line at the end of the file?

Comment (glikson) [inline shared/diveIntoCode/go.md]: No idea - I don't remember touching it. Maybe a vi/git 'side-effect'.

Comment (emirotin) [inline shared/diveIntoCode/go.md]: Actually it is (which is good), but for some reason GH doesn't show it, but an email from it does show.

**Comment by emirotin** [comment]
Thanks for the fix!

**Comment by lurch** [comment]
Should the end of the line also say "remove the line from the `Dockerfile`." instead of "remove the line for the `Dockerfile`."?
Let me know if so and I'll submit a PR.

**Pull request merged**